modules/UI/videolayout/RemoteVideo.js

```diff
@@ -153,7 +152,7 @@ RemoteVideo.prototype._generatePopupContent = function () {
  */
 RemoteVideo.prototype.updateRemoteVideoMenu = function (isMuted, force) {
 
-    this.isMuted = isMuted;
+    this.isVideoMuted = isMuted;
 
```

Comment:
But we call this method from VideoLayout.onAudioMute, so this is not video muted. 
The isMuted was introduced with commit https://github.com/jitsi/jitsi-meet/commit/5654d34ee88e0ec25fbff78c6ece53cabb832646 so I think we just need to rename isMuted to isAudioMuted.